listenbrainz/webserver/static/css/recommendation-page.less

```diff
@@ -0,0 +1,61 @@
+#recommendations {
+  &:extend(#listens);
+  padding: 10px;
+
+  > .recommendation-card {
+    padding: 15px;
+    margin-bottom: 7px;
+    max-height: 72px;
+
+    &.current-recommendation {
+      background-color: rgba(217, 237, 247, 0.3) !important ;
+    }
+
+    > .col-xs-9, .col-xs-11 {
+
+      > .col-xs-9, .col-xs-11 {
```

Comment:
I think you based this styling on what was there before, but a worthy comment:

Among the things that need changing that I mentioned above (for another PR), this use of bootstrap classes that are meant for layout to be used for styling should be replaced.
The reason is that for example if we decide to change the size of the columns but not the styling, we'll have to modify the less rules even though we haven't changed any of the styling.
Instead, we'll want them to be decoupled by using separate classes for styling and layout (like you do below with `.smile, .laugh, .angry, .disappoint, .meh` instead of `> span`).